tests/utils/globalParams.test.ts

```diff
@@ -11,7 +11,7 @@ describe("globalParams", () => {
 
   beforeEach(() => {
     globalParams = dataGenerator.generateGlobalPramsVersions(
-      dataGenerator.getRandomIntegerBetween(1, 100),
+      dataGenerator.getRandomIntegerBetween(2, 100),
```

Comment:
generate al least 2 versions for "should return correct values for params version that are not first or last" test case